packages/tests/shadydom/shady.html

```diff
@@ -1599,6 +1614,23 @@
             ShadyDOM.wrapIfNeeded(el1).remove();
             ShadyDOM.wrapIfNeeded(el2).remove();
           });
+
+          test('elementsFromPoint (noPatch)', function () {
+            const host = createAndPosition(document.body, 0);
+            const shadowRoot = ShadyDOM.wrapIfNeeded(host).attachShadow({
+              mode: 'open',
+            });
+            const inner1 = createAndPosition(shadowRoot, 0);
+            const inner2 = createAndPosition(shadowRoot, 0);
+            const background = [document.body, document.documentElement];
+            ShadyDOM.settings.noPatchDocumentEFP = true;
+            let e = ShadyDOM.wrap(document).elementsFromPoint(5, 5);
+            assert.deepEqual(e, [inner2, inner1, host, ...background]);
```

Comment:
I think `assert.sameOrderedMembers` would be better here and below so that the equality check doesn't crawl down into properties of these nodes. (Maybe it does something special for elements but [the docs for `deepEqual`](https://www.chaijs.com/api/assert/#method_deepequal) basically just say "`deepEqual` means 'deeply equal'" lol.)

Reply:
That doesn't seem to be available and all the tests pass with `deepEqual` so it seems ok to use.

Reply:
Huh, yeah, if this is passing as is then it's probably fine.